archlinuxcn/goldendict-ng/PKGBUILD

```
@@ -2,7 +2,7 @@
 
 pkgname=goldendict-ng
 pkgver=25.10.2
-pkgrel=2
+pkgrel=3
```

```suggestion
pkgrel=2
```
The `pkgrel` doesn't need to be updated manually.